### PR TITLE
FAD-5465 Implements ip pools list view

### PR DESCRIPTION
**Pull request by rajumsys**

### CODE REVIEW

File: src/pages/ipPools/ListPage.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+function mapStateToProps({ ipPools }) {
+  return {
+    ipPools: ipPools.list,
+    loading: ipPools.listLoading,
+    error: ipPools.listError
+  };
+}
+
+export default withRouter(connect(mapStateToProps, { listPools })(IpPoolsList));
```
Comment (jasonrhodes) [inline src/pages/ipPools/ListPage.js]: I don't think you need `withRouter()` here

**Comment by rajumsys** [comment]
Ok, I just realized, this way we are not showing `Default Pool` at all. we'll need to figure out how to show it.

File: src/pages/ipPools/ListPage.js (hunk 1 above)
Comment (rajumsys) [reply src/pages/ipPools/ListPage.js]: Removed

**Comment by rajumsys** [comment]
@jasonrhodes rebased and fixed tests

**Pull request merged**